tests/e2e/Services/Databases/DatabasesBase.php

```diff
@@ -297,6 +297,174 @@ public function testCreateAttributes(array $data): array
+        $this->assertEquals($num_two_hundred['body']['size'], 256);
+        $this->assertEquals($num_two_hundred['body']['required'], false);
+
+        $this->assertEquals(202, $num_million['headers']['status-code']);
+        $this->assertEquals($num_million['body']['key'], '1000000');
+        $this->assertEquals($num_million['body']['type'], 'string');
+        $this->assertEquals($num_million['body']['size'], 256);
+        $this->assertEquals($num_million['body']['required'], false);
+
+        // wait for database worker to create attributes
+        sleep(2);
+
+        $numeric_key_collection = $this->client->call(Client::METHOD_GET, '/databases/' . $databaseId . '/collections/' . $numericKeyId, array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]), []);
+
+        $this->assertIsArray($numeric_key_collection['body']['attributes']);
+        $this->assertCount(3, $numeric_key_collection['body']['attributes']);
+        $this->assertEquals($numeric_key_collection['body']['attributes'][0]['key'], $num_one['body']['key']);
+        $this->assertEquals($numeric_key_collection['body']['attributes'][1]['key'], $num_two_hundred['body']['key']);
+        $this->assertEquals($numeric_key_collection['body']['attributes'][2]['key'], $num_million['body']['key']);
+
+        // Create document with numeric key
+        $document = $this->client->call(Client::METHOD_POST, '/databases/' . $databaseId . '/collections/' . $numericKeyId . '/documents', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()), [
+            'documentId' => ID::unique(),
+            'data' => [
+                '1' => 'one',
+                '200' => 'two hundred',
+                '1000000' => 'one million'
+            ],
+            'permissions' => [
+                Permission::read(Role::user($this->getUser()['$id'])),
+                Permission::update(Role::user($this->getUser()['$id'])),
+                Permission::delete(Role::user($this->getUser()['$id'])),
+            ]
+        ]);
+
+        $id = $document['body']['$id'];
+
+        $this->assertEquals(201, $document['headers']['status-code']);
+        $this->assertEquals($numericKeyId, $document['body']['$collectionId']);
+        $this->assertArrayNotHasKey('$collection', $document['body']);
+        $this->assertEquals($databaseId, $document['body']['$databaseId']);
+        $this->assertEquals($document['body']['1'], 'one');
+        $this->assertEquals($document['body']['200'], 'two hundred');
+        $this->assertEquals($document['body']['1000000'], 'one million');
+        $this->assertIsArray($document['body']['$permissions']);
+        $this->assertCount(3, $document['body']['$permissions']);
```

Comment:
Let's remove these assertions to make the test easier to read, we only need to check the update works okay with the numeric keys